internal/querynodev2/delegator/distribution.go

```diff
@@ -94,9 +93,7 @@ func (q *channelQueryView) Serviceable() bool {
 	// Note: after we support channel level target(data view), we can remove this flag
 	viewReady := q.syncedByCoord
 
-	// if partial result is enabled, we can skip the viewReady check
-	enablePartialResult := paramtable.Get().QueryNodeCfg.PartialResultRequiredDataRatio.GetAsFloat() < 1.0
-	return dataReady && (viewReady || enablePartialResult)
+	return dataReady && viewReady
```

Comment:
This change the definition of serviceable, will it hurt anything? like partial search